lua/lvim/plugins.lua

```diff
@@ -8,21 +8,21 @@ local commit = {
   dapinstall = "568d946a99edb6780912cb39ca68c368516cd853",
   dashboard_nvim = "d82ddae95fd4dc4c3b7bbe87f09b1840fbf20ecb",
   fixcursorhold = "0e4e22d21975da60b0fd2d302285b3b603f9f71e",
-  friendly_snippets = "35bacce3c903dff2852e87a13196cad0dd166093",
-  gitsigns = "f4648dcc055a10573f36e83343592824155ab466",
+  friendly_snippets = "a8efb538f4be58b4f4586bc964f9f10496c05e31",
+  gitsigns = "f43cee333fdfeb27aae86bab73a38c5885d5c7f4",
   lua_dev = "a0ee77789d9948adce64d98700cc90cecaef88d5",
   lualine = "70691ae350fdbe1f15758e3b8e2973742a7967a9",
   luasnip = "0222ee63c9e4b80e6000d064f8efd8edcc6d0c48",
   nlsp_settings = "3a3942b5d1da30e3ca0dc431aada3191c5952054",
-  null_ls = "5213916f51a178287cf5162354f2f7c2c4e204bb",
+  null_ls = "0adceda4fb3fdd25ff015e68ebe176268b12bb2c",
   nvim_autopairs = "97e454ce9b1371373105716d196c1017394bc947",
-  nvim_cmp = "1797f9e1acd2e0b5b4b6805928aebc1dcc0ecbff",
+  nvim_cmp = "7cb14b0c0f926e3cd5ac4d2e19634f82534c9d3c",
   nvim_dap = "c9a58267524f560112ecb6faa36ab2b5bc2f78a3",
-  nvim_lsp_installer = "0f8e6a6557b1e4d685f5ad1bb511be11961e5969",
-  nvim_lspconfig = "58d2ba6b968539a20d701be0bf98ae154456e265",
+  nvim_lsp_installer = "cc34adb58de9e5a42af730c97ec04fc500bb5e63",
+  nvim_lspconfig = "ea29110765cb42e842dc8372c793a6173d89b0c4",
   nvim_notify = "15f52efacd169ea26b0f4070451d3ea53f98cd5a",
-  nvim_tree = "0a2f6b0b6ba558a88c77a6b262af647760e6eca8",
-  nvim_treesitter = "6039e4782a300b0570e54fc7fdcf3b4728203c77",
+  nvim_tree = "2dfed89af7724f9e71d2fdbe3cde791a93e9b9e0",
```

Comment:
This one is gonna be a headache.. 😢 
https://github.com/kyazdani42/nvim-tree.lua/commits/master

I say we leave it alone for now.

Reply:
agreed